apps/dashboard/src/app/(dashboard)/providers.tsx

```diff
@@ -13,11 +14,13 @@ export function AppRouterProviders(props: { children: React.ReactNode }) {
     <QueryClientProvider client={queryClient}>
       <AllChainsProvider>
         <ChainsProvider>
-          <DashboardThirdwebProviderSetup>
-            <ThemeProvider attribute="class" defaultTheme="dark">
-              {props.children}
-            </ThemeProvider>
-          </DashboardThirdwebProviderSetup>
+          <TooltipProvider>
```

Comment:
oof, do we truly need this? why didn't we need it before when we already have tooltips in chainlist etc for example?

Reply:
Oh looks like you made a nice little component that does it all for you, will use that